src/testing/utils.tsx

```diff
@@ -151,3 +155,45 @@ export const renderWrappedHook = <Result, Props>(
   });
   return { router, result, store };
 };
+
+export function createStore(
```

Comment:
A tiny detail, but there's quite a bit of unnecessary whitespace in this function.